mrmustard/lab_dev/circuits.py

```diff
@@ -624,3 +496,120 @@ def component_to_str(comp: CircuitComponent) -> list[str]:
             ret += "\n\n"
 
         return ret
+
+
+def lookup_path(circuit: Circuit) -> None:
```

Comment:
I'm not sure how I feel about it being a standalone method but maybe this should be in a `circuits_utils` folder? We could even include `_indices_dict` there

Reply:
Maybe even `component_to_str` should be in circuit_components_utils

Reply:
The point is that we have a lot of fairly large class methods that maybe make sense to be standalone util methods significantly cleaning up our files